Add k8s docs pages

anthonydillon opened the pull request:
## Done
Use pandoc to convert the [current Kubernetes docs](https://blog.juju.solutions/kubernetes-docs/) into www.ubuntu.com template pages. 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /kubernetes/docs
- Click through the navigation and see the pages load and look good

## Issue / Card
Fixes #4344

webteam-app commented:
Starting demo at: https://www-ubuntu-com-canonical-websites-pr-4355.run.demo.haus/

pmahnke commented on templates/kubernetes/docs/quickstart.html:
```diff
@@ -0,0 +1,148 @@
+</div>
+<p>Use the cursor keys to navigate and <code>space</code> to select either <strong>flannel</strong> <em>or</em> <strong>calico</strong>. If you don't have an opinion about which style of networking you need, just leave the default choice and select <code>next</code>. Some clouds do not currently support Calico, in which case this option will not appear.</p>
+<p>You will now be asked for your <strong>sudo</strong> password. This is so <strong>conjure-up</strong> can download and install the latest version of <code>kubectl</code> (the command line tool for managing Kubernetes) and configure it to work with your new cluster.</p>
+<p>The next screen will show the applications to be deployed. See the <a href="/kubernetes/docs/overview.html">overview</a> if you need a better understanding of what these components do.</p>
```
you need to remove the *.html

pmahnke commented on templates/kubernetes/docs/upgrading.html:
```diff
@@ -0,0 +1,167 @@
+<h3 id="upgrading-the-kubernetes-worker-units">Upgrading the <strong>kubernetes-worker</strong> units</h3>
+<p>For a running cluster, there are two different ways to proceed:</p>
+<ul>
+<li><a href="https://martinfowler.com/bliki/BlueGreenDeployment.html">Blue-green</a> upgrade - This requires more resources, but should ensure a safe, zero-downtime transition of workloads to an updated cluster</li>
```
p-link--external

pmahnke commented on templates/kubernetes/docs/upgrading.html:
```diff
@@ -0,0 +1,167 @@
+<li>The machine from which you will perform the backup has sufficient internet access to retrieve updated software</li>
+<li>Your cluster is running normally</li>
+<li>You read the <a href="/kubernetes/docs/upgrade-notes">Upgrade notes</a> to see if any caveats apply to the versions you are upgrading to/from</li>
+<li>You read the [Release notes][release-notes] for the version you are upgrading to, which will alert you to any important changes to the operation of your cluster</li>
```
should link to /kubernetes/docs/news

pmahnke commented on templates/kubernetes/docs/ceph.html:
````diff
@@ -0,0 +1,83 @@
+ <span class="ex">...and</span> then we<span class="st">&#39;ll add three storage nodes. For the storage nodes, we will also</span>
+<span class="st"> specify some actual  storage for these nodes to use by using `-- storage`. In</span>
+<span class="st"> this case the Juju charm uses labels for different types of storage:</span></code></pre></div>
+<p>juju deploy -n 3 ceph-osd --storage osd-devices=32G,2 --storage osd-journals=8G,1 ```</p>
````
code isn't rendered correctly

anthonydillon commented:
@pmahnke Thanks, I have addressed the comments. Happy to try the markup from Jekyll if it's going to need less processing. These docs are very out of date so will need some work.
